Source/Core/VideoCommon/RenderBase.cpp

```diff
@@ -238,6 +241,8 @@ u16 Renderer::BBoxRead(u32 index)
 
 void Renderer::BBoxWrite(u32 index, u16 value)
 {
+  LWDProfiler::ScopedMarker marker("RenderBase/BBox/Write");
```

Comment:
Cheap, it only marks it as dirty. 

No point tracking this, the actual cost gets pushed into the Flush